src/api/app/assets/javascripts/webui2/application.js

```diff
@@ -22,3 +22,6 @@
 //= require webui2/plotbusyworkers.js
 //= require webui2/datatables.js
 //= require webui2/tabs.js
+//= require webui/application/comment.js
+//= require webui2/request.js
+//= require webui2/jquery-ui.min.js
```

Comment:
For what do we need this?

Reply:
Because you wanted to use jquery3 and for that we need to upgrade the jquery-ui-rails gem to its latest version, and that will affect the old webui needing then to change every single call to the jquery-ui components and also the way it requires the files. That would drive us to many problems and possible issues we won't catch right now.

So, instead of upgrade the jquery-ui-rails gem in the Gemfile we just added the file to the webui2 layout. And whenever we migrate to the webui2 completely we can get rid of this and use the gem in its latest version.